src/viam/examples/modules/tflite/main.cpp

```diff
@@ -71,19 +71,16 @@ class MLModelServiceTFLite : public vsdk::MLModelService {
         // here. It should be safe to tear down all state
         // automatically without needing to wait for anything more to
         // drain.
+        close();
```

Comment:
Please see my comment over in `module/service.cpp`, but I suspect the status quo is that the `close` method was in fact reachable from `ModuleService_`, probably from `RemoveResource`, and maybe from `ReconfigureResource` under some circumstances. I wonder if the move here is to leave this as `close`, and have `MLModelServiceTFLite` derive from `Stoppable` instead. It's a little wrong, in that it doesn't model a "stop GRPC", but at least it keeps the `stop` that the `ModuleService` and `RobotService` want to call on resources wired into the stopp-y behavior of this type.

Reply:
Can we try putting this back to having a `stop` method but having `MLModelServiceTFLite` derive from `Stoppable`? I think that would mean that the `stop_if_stoppable` calls from `ModuleService_` would actually do the right thing, and then we wouldn't need to introduce the new `close` concept.

Reply:
Yep! Derived `MLModelServiceTFLite` from `Stoppable` and maintained its `stop` implementation present on `main`.